diffusion/evaluation/clean_fid_eval.py

```diff
@@ -69,7 +70,8 @@ def __init__(self,
                  output_dir: str = '/tmp/',
                  num_samples: Optional[int] = None,
                  precision: str = 'amp_fp16',
-                 prompts: Optional[List[str]] = None):
+                 prompts: Optional[List[str]] = None,
+                 additional_generate_kwargs: Optional[Dict] = None):
```

Comment:
Nit: any reason you aren't making this `generate_kwargs` and removing arguments from `__init__`? Not a big deal, it can always be done later.

Reply:
The generate args that are taken from `__init__` raw are just the image size. Everything else has to get processed by the evaluator before seeing generate, so I'm inclined to leave the name as `additional_generate_kwargs` to avoid having to override the user specified values there.